transliterate/transliterate.py

```diff
@@ -0,0 +1,38 @@
+from dotenv import load_dotenv
+
+import requests, uuid, json, os
+
+load_dotenv()
```

Comment:
this is fine for now, but it might interfere with how we do deployments once we move to AWS